### ISSUE TITLE
kickgdpr_ga_optout tag in articles

### ISSUE THREAD

**Issue by pulsarinformatique** [Kicktemp/kickgdpr#6]
Hi

Even if I put 

{kickgdpr_ga_optout}Disable Google Analytics{/kickgdpr_ga_optout}

in the legal notices article it only displays the tags without interpretating it 

However I find this behaviour normal since there is no content plugin (only a system plugin)

How do you intend it to mahe it work please ?

Thanks

cyril

**Comment by tecpromotion** [comment]
Hi @pulsarinformatique 
A system plugin always works. https://docs.joomla.org/Plugin/Events/System
No content plugin needed.

Is your legal notice a normal Joomla! item?

**Comment by pulsarinformatique** [comment]
Not, it is a seblod content but the system seblod plugin is always the first one so that all the content  plugins are processed correctly after

Seblod content are joomla extended contents (same #__content table plus additional tables)

**Comment by tecpromotion** [comment]
Unfortunately, we can not rebuild that way. Seblod is just an extension which brings multisite features. As long as the Joomla! Core does not support multisite, it will be difficult to consider all points here.

**Comment by tecpromotion** [comment]
Does the {loadmodule} in a seblod article work for you?

**Comment by pulsarinformatique** [comment]
Yes, just checked again but {kickgdpr_ga_optout}Disable Google Analytics{/kickgdpr_ga_optout} doesn't

cyril

**Comment by pulsarinformatique** [comment]
Theorically you do need to have a content plugin to interpret the {} tags in articles. This is the standard joomla way. If you only have a system plugin it is normal you have cases that break

thanks

cyril

**Comment by nielsnuebel** [comment]
@pulsarinformatique  thanks for the note but that's not correct.

system plugins are always executed after initializing joomla
https://github.com/joomla/joomla-cms/blob/d3fb98cb9ba20a6e03b78ba7ee269b336c04862f/libraries/src/Application/CMSApplication.php#L622

https://github.com/joomla/joomla-cms/blob/d3fb98cb9ba20a6e03b78ba7ee269b336c04862f/components/com_content/views/article/view.html.php#L193

System Plugins are always triggered 
![image](https://user-images.githubusercontent.com/3214077/45802228-8f5b5f80-bcb6-11e8-9841-3e0c564f33e9.png)

![debug](https://user-images.githubusercontent.com/3214077/45803031-bfa3fd80-bcb8-11e8-8719-3c9428ef9282.gif)

Maybe Seblod grab introtext and not text.

**Comment by pulsarinformatique** [comment]
yes the problem is that SEBLOD puts stuff in the introtext and uses a special CONTENT plugin (that should be the first on the list) to hide his own stuff.

in our example, if I deactivate the seblod content plugin your system plugin works but of course I can't deactivate the seblod content plugin :(

So far we never faced such a problem because all third party extensions that put some {} tags in the article used content plugins and not system plugins.

thanks
